virttest/qemu_vm.py

```diff
@@ -2305,6 +2305,9 @@ def sort_key(dev):
             options["deflate-on-oom"] = deflate_on_oom
             guest_polling = balloon_params.get("balloon_opt_guest_polling")
             options["guest-stats-polling-interval"] = guest_polling
+            balloon_addr = balloon_params.get("balloon_addr")
+            if balloon_addr:
+                options["addr"] = balloon_addr
```

Comment:
That's really a bad workaround, but since we can not expect migration support been fixed recently, I will accept it, by requiring a comment to declare this is only a workaround for migration, do not recommend to use it other way.

Reply:
@vivianQizhu If you were saying the referenced tp-qemu patch, I wouldn't argue for it.  But for this one, it implements the addr option for balloon device, just like other devices. I won't say it is a workaround.  Thanks.

Reply:
Agree with `balloon_pci_addr` is better, and no need to add comment here :)

Reply:
@vivianQizhu @yanan-fu Thanks for your review.  I guess there might be some negative tests in the future which need setting addr manually.  It's not a must to set it manually, so I think it won't do any harm. As for the variant name, I will update when we have a final conclusion about this patch. Thanks.